react-native-cli/index.js

```diff
@@ -94,8 +94,16 @@ function init(name) {
       process.exit(1);
     }
 
-    var cli = require(CLI_MODULE_PATH());
-    cli.init(root, projectName);
+    run('npm install --save flow-bin@0.10.0', function(e) {
```

Comment:
You don't want to change react-native-cli. This is installed via npm install -g and should never change. You want to update 

https://github.com/facebook/react-native/blob/master/cli.js